clients/kafka-common/src/main/java/org/eclipse/hono/kafka/client/tracing/LongTag.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package org.eclipse.hono.kafka.client.tracing;
+
+import io.opentracing.Span;
+import io.opentracing.tag.AbstractTag;
+
+/**
+ * An OpenTracing tag type for {@code long} values.
+ */
+public class LongTag extends AbstractTag<Long> {
```

Comment:
this doesn't seem to be kafka specific, is it?

Reply:
No, it's not. Let's decide on [how to proceed](https://github.com/eclipse/hono/pull/2303#discussion_r526028936) with tracing, then I will move it to a more appropriate location.

Reply:
I do not see any reason why we shouldn't move it to the place where our other OpenTracing related (helper) classes are currently located, or is there? FMPOV this should go into `org.eclipse.hono.tracing` in the hono-core module ...

Reply:
I agree. The only reason I did not do it yet is that we discussed if we shouldn't drop the tags for the `RecordMetadata` and if we do so, I would remove the `LongTag` as it is not used anywhere else.